app/root.css

```diff
@@ -8,32 +8,36 @@
 :root.light {
   /* light theme: */
   --bgColorPage: #eee;
-  --bgColorQuestionTitle: #bdf;
-  --bgColorQuestionAnswer: #eff;
+  --bgColorQuestion: #f0f2f6;
   --bgColorCopied: #fed;
   --colorText: #222;
   --colorTitle: #666;
   --colorTitleHighlight: #d86;
   --colorQuestionTitle: #333;
+  --colorButtonText: #3073b7;
+  --colorTag: #3073b7;
   --borderColor: #ddd;
   --borderColorButton: #999;
-  --bgColorInput: #fff;
-  --bgColorButton: #ddd;
+  --bgColorButton: #dcdcdc;
   --bgColorHighlight: #c0d3e4;
   --bgColorTableRows: #e8f7fe;
   --colorLink: #79f;
   --colorLinkVisited: #8ac;
-  --colorTooltip: #d1d1d1;
+  --colorTooltip: rgba(48, 115, 183, 0.1);
```

Comment:
* [ ] could you please find a better background for `.answer .link-popup`? transparent `--colorTooltip` is no good for that:
  ![image](https://github.com/StampyAI/stampy-ui/assets/1087670/b71a7abe-f98d-4004-affb-697f77b2771d)

@ccstan99 - I didn't find the popup styles in Figma - I'm happy if @Mink-Mink will pick from existing bg colors unless you have some new color in mind and just forgot to specify it yet

Reply:
No transparency please. No new color. Feel free to pick from the existing bg color.